packages/react-examples/src/react-experiments/Persona/Persona.Example.tsx

```diff
@@ -25,6 +25,11 @@ export class PersonaExample extends React.Component<{}, {}> {
             <Stack tokens={tokens.personaCoinStack}>
```

Comment:
there's a version of this in the main react directory, that should be updated as well (and I think there are snapshot tests for that one)